tools/extract_images.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
```

Comment:
Doesn't belong here

Reply:
oops... this is what you get from git commit -am